bugbot/rules/not_landed.py

```diff
@@ -340,7 +321,6 @@ def get_bugs(self, date="today", bug_ids=[]):
 
         bz_reviewers = self.get_bz_userid(reviewers_phid)
         all_reviewers = set(bz_reviewers.keys())
-        nicknames = self.get_nicks(nicknames)
```

Comment:
Since we are not querying nicknames here anymore, we may want to change the name from `nicknames` to something more representing the content, e.g., `bug_assignee_map`.

Reply:
changed